wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -345,6 +345,65 @@ final class ConversationEventPayloadProcessor {
         _ = ZMSystemMessage.createOrUpdate(from: originalEvent, in: context)
     }
 
+    // MARK: - Protocol Change
+
+    func processPayload(
+        _ payload: Payload.ConversationEvent<Payload.UpdateConversationProtocolChange>,
+        originalEvent: ZMUpdateEvent,
+        in context: NSManagedObjectContext
+    ) async {
+        guard let qualifiedID = payload.qualifiedID else {
+            Logging.eventProcessing.error("processPayload of event type \(originalEvent.type): Conversation qualifiedID missing, aborting...")
+            return
+        }
+
+        await syncProtocolChangeConversation(qualifiedID, in: context.notificationContext)
+
+        addProtocolChangeSystemMessage(
+            qualifiedID,
+            messageProtocolRawValue: payload.data.messageProtocol,
+            in: context
+        )
+    }
+
+    private func syncProtocolChangeConversation(_ qualifiedID: QualifiedID, in notificationContext: NotificationContext) async {
```

Comment:
Just thinking this could be a shorter name and still maintain clarity about what the method is doing.
```suggestion
    private func syncConversation(_ qualifiedID: QualifiedID, in notificationContext: NotificationContext) async {
```